python/ray/data/tests/test_dataset_formats.py

```diff
@@ -2903,6 +2903,23 @@ def get_node_id():
     assert sorted(ds.take()) == ["goodbye", "hello", "world"]
 
 
+def test_read_s3_file_error(ray_start_regular_shared, s3_path):
+    dummy_path = s3_path + "_dummy"
```

Comment:
So this is not an error like "not found" ?

Reply:
file not found and missing credential have same error. File not found is easier to unit test so doing test for it here.